src/webnn_native/dml/deps/src/device.h

```diff
@@ -19,7 +19,7 @@ namespace pydml
     class Device
     {
     public:
-        explicit Device(bool useGpu = true, bool useDebugLayer = false);
+        explicit Device(bool useGpu = true, bool useDebugLayer = false, DXGI_GPU_PREFERENCE gpuPreference = DXGI_GPU_PREFERENCE::DXGI_GPU_PREFERENCE_UNSPECIFIED);

```

Comment:
The gpu prefernce is unnecessary for using cpu, so it's better to add a method for the `Device ` to set the preference.

Reply:
Thanks! I will consider and also submit a PR to DirectML github to update the [device.cpp](https://github.com/microsoft/DirectML/blob/master/Python/src/device.cpp) directly, hope that we can get more feedbacks. @fujunwei

Reply:
Good point. @mingmingtasd , could you please file an issue to track it.